.github/workflows/linux.yml

```diff
@@ -194,9 +195,11 @@ jobs:
               -D DEAL_II_EARLY_DEPRECATIONS=ON \
               -D DEAL_II_WITH_MPI=ON \
               -D DEAL_II_WITH_LAPACK=ON \
+              -D DEAL_II_WITH_TBB=ON \
               -D MPI_DIR=${I_MPI_ROOT} \
               -D BLAS_DIR=${MKLROOT} \
               -D LAPACK_DIR=${MKLROOT} \
+              -D TBB_DIR=${TBBROOT} \
```

Comment:
Is that something that's set in the OS?

Reply:
The command `source /opt/intel/oneapi/setvars.sh` sets all relevant environment variables, i.e., `I_MPI_ROOT`, `MKLROOT`, and `TBBROOT`.

See also https://www.intel.com/content/www/us/en/develop/documentation/get-started-with-intel-oneapi-dlfd-linux/top/before-you-begin.html